SnapChat/App/Components/AddFriends/AddFriends.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react'
+import { View, Text, TouchableHighlight, Image, ScrollView } from 'react-native'
+import { addFriendsStyles as styles } from './addFriendsStyles'
+
+const AddFriends = ({backButtonPressed}) => (
+
+    <View style={styles.container}>
+
+        {/* Header */}
+        <View style={styles.header}>
+            <TouchableHighlight
+                onPress={backButtonPressed}
+                underlayColor="#F5F5F5">
+                <Image
+                    style={styles.backArrow}
+                    source={require('../../../images/back_arrow.png')}
+                />
+            </TouchableHighlight>
+            <Text style={styles.headerTitle}>Add Friends</Text>
+            <Text style={styles.backArrow}></Text>
+        </View>
+
+        <ScrollView>
+            <Row field="Add by Username" />
+            <Row field="Add from Address Book" />
+            <Row field="Add by Snapcode" />
+            <Row field="Add Nearby" />
+            <Row field="Share username" />
+        </ScrollView>
+
+    </View>
+
+)
+
+const Row = ({field}) => {
+    return (
+        <View>
+            <View style={styles.fieldInfo}>
+                <Text style={styles.fieldTitle}>{field}</Text>
+                <Image
+                    style={styles.addedMeImg}
+                    source={require('../../../images/forward_arrow.png')}
+                />
+            </View>
+        </View>
```

Comment:
Okay @rokane . What the github code review is confusing. Before i clicked on line 37 because thats what i wanted to comment on but it only shows the code above. Anyway to the issue

So the Row component, it returns the view component and then straight away inside of it another view component. So it's just an extra wrapper that isn't needed. It wouldn't be an issue here, but if all the code had extra wrappers extra rendering all of them could become bloated/slow. 

So instead of 
<View>
  <View style={..}>
   {..code..}
  </View>
</View>

to
<View style={}>
  {..code}
</View>